README.md

```diff
@@ -0,0 +1,111 @@
+<p align="center">
+	<img src="docs/eoslocal.png" width="600">
+</p>
+
+<p align="center">
+	<a href="#">
+		<img src="https://img.shields.io/dub/l/vibe-d.svg" alt="MIT">
+	</a>
+</p>
+
+# EOS Mainnet Local Network
+
+## Description
+EOS Local provides a quick way to setup an EOS local network for development.
+
+The primary benefits of containers are consistency across different environments and deployment ease.
+
+This project has a [full-stack-boilerplate](https://github.com/eoscostarica/full-stack-boilerplate) for your dApp. 
+
+### Why to use a local environment?
+Having a local environment provides a series of benefits that you cannot in a public network, for example, with EOS Mainnet locally, transaction costs are avoided since they are carried out in a development environment and not in production, also, they are accessed to system contracts to modify them as appropriate.
+
+In a Blockchain network every transaction creates an immutable record and everything that is modified can affect both positively and negatively the users within it, it is for this reason that it is essential to have an environment premises where functionality tests, performance tests, stress tests, among others, can be carried out without the risk of producing a failure that affects users.
+
+Finally, a factor to consider is the time that is reduced in the initial configuration of any network, this image allows directly, with only two commands to have the network installed and ready to perform functionality tests as necessary.
+
+### Contracts
+The EOSIO image is based on the `eosio.system`, `eosio.token` and `eosio.msig` contracts for its configuration. Your code can be found at [this link](https://github.com/EOSIO/eosio.contracts/tree/master/contracts).
+1. **eosio.system**: Defines the structures and actions needed for blockchain's core functionality.
+2. **eosio.token**: Defines the structures and actions that allow users to create, issue, and manage tokens for EOSIO-based blockchains.
+3. **eosio.msig**: Allows the creation of proposed transactions that require authorization from a list of accounts.
+
```

Comment:
should we load the eden contracts also?
simpleassets / atomicassets/ o dgoods ?

Reply:
I don't thinks so but we could create a section next to it talking about some projects that can be deployed on it

Reply:
Fixed here: https://github.com/eoscostarica/eos-local/pull/88/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R33